MEME/MEME/Model/ModelHomeView/View/ModelHomeViewController.swift

```diff
@@ -0,0 +1,403 @@
+        contentsView.addSubview(modelWelcomeGuideLabel)
+        contentsView.addSubview(viewAllReservationsButton)
+        contentsView.addSubview(collectionView)
+        scrollView.addSubview(contentsView)
+        view.addSubview(scrollView)
+    }
+    
+    // MARK: - makeConstraints
+    func makeConstraints() {
+        scrollView.snp.makeConstraints { (make) in
+            make.edges.equalTo(view.safeAreaLayoutGuide)
+        }
+        contentsView.snp.makeConstraints { (make) in
+            make.edges.equalTo(scrollView)
+            make.width.equalTo(scrollView)
+        }
+        memeLogoImageView.snp.makeConstraints {make in
+            make.top.equalTo(contentsView.snp.top).offset(10)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+            make.height.equalTo(42)
+            make.width.equalTo(memeLogoImageView.snp.height).multipliedBy(67.0/42.0)
+        }
+        alarmImageView.snp.makeConstraints {make in
+            make.centerY.equalTo(memeLogoImageView.snp.centerY)
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+            make.height.equalTo(22)
+            make.width.equalTo(memeLogoImageView.snp.height).multipliedBy(6.0/7.0)
+        }
+        searchMakeup.snp.makeConstraints {make in
+            make.top.equalTo(memeLogoImageView.snp.bottom).offset(27)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+            make.height.equalTo(41)
+        }
+        modelWelcomeLabel.snp.makeConstraints {make in
+            make.top.equalTo(searchMakeup.snp.bottom).offset(33)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+        }
+        modelWelcomeGuideLabel.snp.makeConstraints {make in
+            make.top.equalTo(modelWelcomeLabel.snp.bottom)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+        }
+        viewAllReservationsButton.snp.makeConstraints {make in
+            make.top.equalTo(modelWelcomeGuideLabel.snp.bottom).offset(17)
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+        }
+        collectionView.snp.makeConstraints {make in
+            make.top.equalTo(viewAllReservationsButton.snp.bottom).offset(13)
+            make.height.equalTo(800)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+            make.bottom.equalTo(contentsView.snp.bottom)
+    
+        }
+    }
+    
+    // MARK: - BindViewModel
+    private func bindViewModel() {
+        let memberId = KeyChainManager.loadMemberID()
+        
+        let input = ModelHomeViewModel.Input(
+            modelReservationsTrigger: Observable.just(memberId),
+            getRecommendArtistByReviewTrigger: .just(()),
+            getRecommendArtistByRecentTrigger: .just(())
+        )
```

Comment:
```suggestion
        let input = ModelHomeViewModel.Input(
            modelReservationsTrigger: Observable.just(memberId),
            getRecommendArtistByReviewTrigger: .just(()),
            getRecommendArtistByRecentTrigger: .just(()))
```

이 부분 제가 컨벤션을 잘못 알고 있었는데요, 위와 같은 형식이 저희가 사용하기로 한 컨벤션입니다. 혼란을 드려 죄송합니다 ㅠㅠ

또 위와 같이 구현하면 ModelHomeViewController의 cell들이 한 번 표시되고 나면 새로고침이 이뤄지지 않을 것 같습니다.
추후 UIViewController를 생명주기와 RxSwift를 결합해 생명주기함수를 쉽게 사용할 수 있게 하는 syntax sugar 코드 추가하겠습니다! 추가 후에 viewWillAppear가 일어날때 이벤트를 input으로 전달하면 좋을 것 같습니다.